src/scripts/create_dummy_reporting_data.py

```diff
@@ -188,8 +188,8 @@ def generate_webhooks_for_target(
 ):
     """Generate Webhooks."""
     if decpection_rate < 0:
-        random.seed()
-        decpection_rate = random.random()
+        random.seed()  # nosec
+        decpection_rate = random.random()  # nosec
```

Comment:
BTW, you have a typo in this variable name: `decpection_rate` --> `deception_rate` - make sure you change it throughout the file.

Reply:
Ha!  I thought "decpectation" was a word I didn't know.

Reply:
@dav3r Fixed that typo.